### PR TITLE
Page speed insights modifications

### PR DESCRIPTION
**Pull request by Etiene**
### Motivation

- Improving page load speed by following the advices of Google insights. 
- The PR was created so I can analyse the staging environment too

### Test plan

- TODO: Write a test plan.

### Pre-merge checklist

- [ ] Documentation
- [ ] Unit tests
- [ ] Reviews
  - [ ] Code review
  - [ ] Design review
- [ ] Manual testing
  - [ ] Windows 7 IE 11
  - [ ] Windows 10 Edge
  - [ ] Mac OS El Capitan Safari
  - [ ] Mac OS El Capitan Chrome
  - [ ] Mac OS El Capitan Firefox
  - [ ] iOS 9 Safari
  - [ ] Android 6 Chrome
  - [ ] Listen to the site on a screen reader
  - [ ] Navigate the site using the keyboard only
- [ ] Tester approved

### CODE REVIEW

File: site/slices/contact-us-slice/form/index.js [hunk 1]
```diff
@@ -38,7 +38,7 @@ class Form extends Component {
         </h2>
 
         <form className={styles.contactUsForm}>
-          <label className={styles.formLabel} htmlFor="message">
+          <label className={styles.formLabel} htmlFor="contactUsMessage">
```
Comment (asavin) [inline site/slices/contact-us-slice/form/index.js]: Were you going to change the name of the text area element below to `contactUsMessage` ?

**Comment by asavin** [comment]
Looks good!

File: site/slices/contact-us-slice/form/index.js (hunk 1 above)
Comment (Etiene) [inline site/slices/contact-us-slice/form/index.js]: The `id` of it is "contactUsMessage". The `htmlFor` is targeting the `id` and not the `name`, so I changed the `htmlFor` to the `id`.

**Comment by gregdorward** [comment]
Tests completed so far:

- Side by side comparison with the current production site - general sanity check and ensuring all images are still in place

- Ran Google Page Speed Insights against Staging and Production - Staging showed a performance improvement compared to what is currently in Production

- Verified that the Contact Us section has the correct labels associated with the forms

I'll pick up where I left off in the morning...

**Pull request merged**